### DISCUSSION TITLE
Handling of new user accounts, relates to issue #24

### DISCUSSION THREAD

**Discussion by krotz-dieter** [Siemens-Healthineers/K2s#26] [Ideas]
The cluster config, setup.json, ... all files are currently available under the user who is setting up the cluster.
Would like to keep it this way, the user is responsible for the cluster and entire setup (from security pint of view).
For sharing with other users i would propose a new feature as a new cmd:
k2s system user ...
which provides the config files as well as ensures the PATH entries also for the other user.
In this way we could also generate a new kubeconfig with other content specific for different use cases.

**Comment by mahenarayan** [comment]
Yes I agree, we should not handle config files, this is user's responsibility to ensure k2s is installed and operated under same user. Makes sense only to provide command which displays basic information about installed setup. Or we can add this in k2s status as well.
> k2s system user 

or 

> k2s status

**Comment by jgriegershs** [comment]
After having an online discussion, we agreed that:
- first step/increment is to grant the same access rights the installing user has to the new user account in question, fine-grained permissions are out of scope for now -> create new command, e.g. `k2s system user add`, `k2s system user ls` and `k2s system user rm`
- `.kube` folder must be re-created for additional users, containing their K8s auth certs
- `.ssh` folder must be re-created for additional users, containing their SSH auth key
- `setup.json` file must be moved to a system-central location, write access only granted to the installing user and read access for all others
- `.docker` folder containing credentials for access of `shsk2s.azurecr.io` registry -> usage of `config.json` file must be analyzed; ideally this file can be removed and the *Docker* daemon be configured in a different way
